geniza/corpus/management/commands/export_metadata.py

```diff
@@ -22,6 +26,16 @@ class MetadataExportRepo(Timerable):
     repo_dir_data = "data"
     ext_csv = ".csv"
 
+    #: default commit message
+    default_commit_msg = "Automated metadata export from PGP"
+
+    exports = {
+        "documents": PublicDocumentExporter,
+        "fragments": PublicFragmentExporter,
+        "sources": SourceExporter,
+        "footnotes": FootnoteExporter,
+    }
```

Comment:
Nice! We have enough now it's good we're wrangling them together programmatically.